var/spack/repos/builtin/packages/libspatialite/package.py

```diff
@@ -18,6 +18,7 @@ class Libspatialite(AutotoolsPackage):
 
     license("MPL-1.1")
 
+    version("5.1.0", sha256="43be2dd349daffe016dd1400c5d11285828c22fea35ca5109f21f3ed50605080")
     version("5.0.1", sha256="eecbc94311c78012d059ebc0fae86ea5ef6eecb13303e6e82b3753c1b3409e98")
     version("5.0.0", sha256="7b7fd70243f5a0b175696d87c46dde0ace030eacc27f39241c24bac5dfac6dac")
     # Must download manually from:
```

Comment:
Any reason we shouldn't deprecate all of these weird/alpha/beta versions? I don't see any packages that require them.

Reply:
fixed! 4.3.0a was released after 4.3.0. so let's keep it.

Reply:
Could still deprecate those ancient versions if we don't want to support them, up to you

Reply:
version 5.0.0.2.c7f67038bf isn't available on downloads. So, I'll remove it without deprecate notice.